src/less/tab/ds6/tab.less

```diff
@@ -10,14 +10,30 @@
     border-bottom: 1px solid @ds6-color-white; // prevent margin-collapse on next element
 }
 
+// move the thin grey bar up by 1 pixel
+.tabs__content,
+.fake-tabs__content {
+    border-top: 1px solid #c7c7c7;
```

Comment:
This grey is part of that [updated set we learned about this morning](https://github.com/eBay/skin/issues/580), and we do not yet have a variable for it. 😞

Reply:
Want me to take care of that task then? I thought you were doing it.

Reply:
Oh, scrap that. We can't because it's a breaking change.